src/web/components/ParticipantRequests/ParticipantApprovalForm.tsx

```diff
@@ -94,6 +145,23 @@ function ParticipantApprovalForm({
                 value: p.id,
               }))}
             />
+            <Input inputName='readonly-requestor-name' label='Requestor Name'>
+              <input
+                className='input-container'
+                disabled
+                value={participant.requestingUser.fullName}
+              />
+            </Input>
+            <Input inputName='readonly-requestor-email' label='Requestor Email'>
```

Comment:
Can we use `<TextInput inputName='readonly-requestor-email' label='Requestor Email' value={participant.requestingUser.email} disabled/>` instead?

Reply:
TextInput attaches a controller and tries to make it part of the form, but this is only for display purposes - I didn't want to use `Input` either but it gives me the label and some other things.